.github/workflows/qa.yml

```diff
@@ -0,0 +1,47 @@
+name: Run QA Framework tests on Github Actions
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      firefox:
+        image: selenium/node-firefox:3.141.59-gold
+        env:
+          HUB_HOST: hub
+          HUB_PORT: 4444
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Set up Maven
+        uses: aahmed-se/setup-maven@v3
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
+        with:
+          maven-version: 3.6.1
```

Comment:
Do we explicitly need to set the Maven version? Otherwise, the GitHub actions images already have Maven installed.

Reply:
that's awesome, i didn't imagine it's installed by default and didn't want to end up with maven 2